migrations.lock

```diff
@@ -147,6 +149,8 @@ blobs
  0009_delete_blobexpiration
  0010_auto_20191023_0938
  0011_blobmeta_compressed
+captcha
+ 0001_initial
```

Comment:
This probably got re-introduced from out-of-date packages.  Running a `pip-sync` and then `./manage.py makemigrations --lock-update` should resolve it.

Reply:
done